akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/TimerScheduler.scala

```diff
@@ -1,6 +1,7 @@
 /**
  * Copyright (C) 2017-2018 Lightbend Inc. <https://www.lightbend.com>
  */
+
```

Comment:
do we have to add these newlines?

Reply:
In my opinion that looks consistent with `akka-http`

Reply:
They're also there in `akka-http` indeed, but they do look a bit strange to me as well. Do we want them?